src/models/callercalleeproxy.h

```diff
@@ -44,5 +40,16 @@ class CallerCalleeProxy : public QSortFilterProxyModel
         return CallerCalleeProxyDetail::match(this, key);
     }
 
+    bool lessThan(const QModelIndex& source_left, const QModelIndex& source_right) const override
+    {
+        QVariant left = source_left.data(sortRole());
```

Comment:
I would change the logic: check the column first like is done in the model itself and then only if that's matching request the variants and assert that `canConvert` is true